button.js

```diff
@@ -119,14 +158,24 @@ async function moveToBugzilla(target) {
   const product = productDropdown.value;
 
   if (!component || !product) {
-    componentDropdown.style.display = 'initial';
-    productDropdown.style.display = 'initial';
+    for (dropdown of allDropdowns) {
+      dropdown.style.display = 'initial';
+    }
   } else {
-    browser.runtime.sendMessage({
+      const severity = severityDropdown.value;
```

Comment:
What are the default values for severity and priority? What happens if they are `undefined`?

To me, this is the most concerning change in this patch. We, for example, do our triage in Bugzilla, by looking at bugs individually. Is this meant for a bulk triage without looking at the bugs?

Reply:
They don't get a value, so yeah, the bot ends up nagging about them. Watching Chris Peterson fall into this exact problem was why I added these fields.